src/typescript/fable-splitter/index.ts

```diff
@@ -0,0 +1,172 @@
+import * as Babel from "babel-core";
+import * as fs from "fs";
+import * as Path from "path";
+import * as Process from "process";
+const client = require("fable-utils/client");
+const babelPlugins = require("fable-utils/babel-plugins");
+
+const customPlugins = [
+    babelPlugins.getRemoveUnneededNulls(),
+    babelPlugins.getTransformMacroExpressions(Babel.template),
+];
+
+const DEFAULT_PORT = parseInt(process.env.FABLE_SERVER_PORT || "61225", 10);
+const FSHARP_EXT = /\.(fs|fsx|fsproj)$/;
+const FSPROJ_EXT = /\.fsproj$/;
+const JAVASCRIPT_EXT = /\.js$/;
+
+let alreadyCompiled: Set<string>;
+
+export type FableOptions = {
+    path?: string;
+    define?: string[];
+    plugins?: string[];
+    fableCore?: string;
+    declaration?: boolean;
+    typedArrays?: boolean;
+    clampByteArrays?: boolean;
+    extra?: any;
+};
```

Comment:
We need to add an option here to change the extension of the F# files, at the moment I think they remain ".fs" in the imports.